ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayDeviceLocationWithNmeaDataSources/README.md

```diff
@@ -0,0 +1,34 @@
+# Display device location with NMEA data sources
+
+This sample demonstrates how to parse NMEA sentences and use the results to show device location on the map.
+
+![](screenshot.png)
+
+## Use case
+
+NMEA sentences can be retrieved from GPS receivers and parsed into a series of coordinates with additional information. Devices without a built-in GPS receiver can retrieve NMEA sentences by using a separate GPS dongle, commonly connected bluetooth or through a serial port.
+
+The NMEA location data source allows for detailed interrogation of the information coming from the GPS receiver. For example, allowing you to report the number of satellites in view.
+
+## How to use the sample
+Tap "Start" to parse the NMEA sentences into a simulated location data source, and initiate the location display. Tap "Reset" to reset the location display.
+
+## How it works
+
+1. Load NMEA sentences from a local file.
+2. Parse the NMEA sentence strings, and push data into `NMEALocationDataSource`.
+3. Set the `NMEALocationDataSource` to the location display's data source.
```

Comment:
```suggestion
2. Parse the NMEA sentence strings, and push data into `NmeaLocationDataSource`.
3. Set the `NmeaLocationDataSource` to the location display's data source.
```

Reply:
There are more throughout this readme and the QML version but I won't mark them all